src/pages/index.tsx

```diff
@@ -20,21 +21,20 @@ export default function DashboardPage({ liveStake }: Props): React.ReactElement
   }
 
   function compareFn(a: Partner, b: Partner): number {
-    const x = liveStake[a.id];
-    const y = liveStake[b.id];
+    const x = liveStake[a.ticker];
+    const y = liveStake[b.ticker];
     if (x === undefined || y === undefined) {
       return 0;
     }
     return x - y;
   }
 
   const sortedPartner: Partner[] = PARTNERS.sort(compareFn);
-  const totalStake: number = Object.values(liveStake).reduce((sum, x) => sum + x, 0);
 
   return (
     <Layout>
       <div className="flex-1 w-full px-2 lg:w-auto lg:pl-8 lg:px-0">
-        <div className="px-4 sm:px-8 flex flex-col overflow-y-scroll bg-white divide-y shadow-xl max-h-[85vh] divide-opacity-10 divide-secondary rounded-3xl">
+        <div className="px-4 sm:px-8 flex flex-col overflow-y-scroll bg-white divide-y shadow-xl max-h-[84vh] divide-opacity-10 divide-secondary rounded-3xl">
           <div className={'sticky top-0 left-0 right-0 z-10 pt-5 space-y-2 bg-white xl:space-y-0 pb-7'}>
             <div className="flex flex-col">
               <div className="text-lg font-bold">Smallest pool now (25% bonus): {sortedPartner[0]?.ticker}</div>
```

Comment:
"Live stake data is pulled from pool.pm API so there might be a time lag. In case of pools having live
                stake close to each other, it is best to consult other websites before staking." => cái description này nếu `source` là Minswap thì ẩn đi, còn nếu `source` là Blockfrost thì hiện là "Because Minswap server is down, data is fetched from Blockfrost instead."